integrations/airflow/marquez_airflow/extractors/__init__.py

```diff
@@ -166,12 +176,12 @@ def validate(self):
         assert (self.operator_class is not None and
                 self.operator.__class__ == self.operator_class)
 
-    # TODO: Only return a single StepMetadata object.
     @abstractmethod
-    def extract(self) -> [StepMetadata]:
+    def extract(self) -> Union[StepMetadata, List[StepMetadata]]:
+        # Strongly preferred to return single StepMetadata instance
```

Comment:
In future releases, we'll want to deprecate returning a list of `StepMetadata` and simply return a `StepMetadata` object.

Reply:
Nice wording, changed :+1: